Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsActivity.java

```diff
@@ -211,11 +218,19 @@ public void showLoading()
   {
     if (dialog == null)
     {
-      dialog = new ProgressDialog(this);
-      dialog.setMessage(getString(R.string.load_settings));
-      dialog.setIndeterminate(true);
+      ProgressBar progressBar =
+              new ProgressBar(this, null, android.R.attr.progressBarStyle);
```

Comment:
What does passing `android.R.attr.progressBarStyle` actually do here?

Reply:
I believe I made a mistake where I thought I was changing the theme from the indeterminate progress bar to the circular version. Additionally, there is a more appropriate material 3 provided component I'll use to replace this